app/sidekiq/lighthouse/poll_form526_pdf.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require 'lighthouse/benefits_claims/service'
+require 'sentry_logging'
+require 'logging/third_party_transaction'
+require 'sidekiq/form526_job_status_tracker/job_tracker'
+require 'sidekiq/form526_job_status_tracker/metrics'
+
+module Lighthouse
+  class PollForm526Pdf
+    include Sidekiq::Job
+    include Sidekiq::Form526JobStatusTracker::JobTracker
+    extend Logging::ThirdPartyTransaction::MethodWrapper
+    extend ActiveSupport::Concern
+
+    attr_accessor :submission_id
+
+    # Sidekiq has built in exponential back-off functionality for retries
+    # A max retry attempt of 15 will result in a run time of ~36 hours
+    # Changed from 15 -> 14 ~ Jan 19, 2023
+    # This change reduces the run-time from ~36 hours to ~24 hours
```

Comment:
Comment should either be modified or removed.

Reply:
thanks. killed it.